chai-dom.js

```diff
@@ -137,6 +141,33 @@
     }
   })
 
+  chai.Assertion.overwriteChainableMethod('length',
+    function(_super) {
+      return function(length) {
+        var obj = flag(this, 'object');
+        var actualLength = obj.children ? obj.children.length : obj.length;
+        if (actualLength) {
+          var desc = obj.children ? elToString(obj) : nodeListToString(obj);
```

Comment:
`elToString` should just handle both cases I think. Then all the other validators get the same benefit. That impacts all the other tests and I can do that as a 2nd pass. Love it though.